### ISSUE TITLE
initMux triggers a crash on iOS

### ISSUE THREAD

**Issue by peterfajner-ensemble** [THEOplayer/react-native-connectors#200]
Hi, I've added the barebones example from the docs to my app:

```
  const [mux, initMux] = useMux({
    debug: true,
    data: {
      env_key: '[my key]', // required
      // Site Metadata
      viewer_user_id: '12345',
      experiment_name: 'player_test_A',
      sub_property_id: 'cus-1',
      // Player Metadata
      player_name: 'My Main Player',
      player_version: '1.0.0',
      player_init_time: 1451606400000,
      // Video Metadata
      video_id: 'abcd123',
      video_title: 'My Great Video',
      video_series: 'Weekly Great Videos',
      video_duration: 12000, // in milliseconds
      video_stream_type: 'on-demand', // 'live' or 'on-demand'
      video_cdn: 'Akamai'
    }
  });

  const PlayerComponent = () => {
    // AudioVideoPlayer returns a THEOplayerView
    return (
      <AudioVideoPlayer
        onPlayerReady={player => {
          initMux(player);
        }}
      />
    );
  };
```

On startup, the app immediately crashes. XCode pauses execution at this line:

![Screenshot 2024-07-24 at 13 57 56](https://github.com/user-attachments/assets/397bbc6b-04c3-4a98-b07f-22c31e1eeffb)

yarn.lock:

```
"@mux/mux-data-theoplayer@^4.17.1":
  version "4.17.1"
  resolved "https://registry.yarnpkg.com/@mux/mux-data-theoplayer/-/mux-data-theoplayer-4.17.1.tgz#a75677b7b77cbd0c2c66f22dd72d9d33a6868239"
  integrity sha512-0cL5SqkGFp90DVvriAfMs/Ptq6B6+QHbRBxUVVE7Q52k/WN3v6JJMMGtAtSBR9kanPM/bCJOnhlW5lLP1J+DaA==
  dependencies:
    mux-embed "4.27.0"

mux-embed@4.27.0:
  version "4.27.0"
  resolved "https://registry.yarnpkg.com/mux-embed/-/mux-embed-4.27.0.tgz#a0e8ce6e2b1025e2304723c7efd00f45b4842a5e"
  integrity sha512-LZ790wIgmFZebOcTqORi85d8r+aNCFcRvG0Wbc4wT7cEeOPn2Pb1iEDSFpJ+3DsgVwxQ38/e9Nwmdd/9hrfZCQ==

"@theoplayer/react-native-analytics-mux@^1.3.0":
  version "1.3.0"
  resolved "https://registry.yarnpkg.com/@theoplayer/react-native-analytics-mux/-/react-native-analytics-mux-1.3.0.tgz#0ae26779d012a66113abbef033e0c691fda8e3ae"
  integrity sha512-cOr5dqO3P/i8vUDIOY8KLRVCOk6qYID5h4RiPqEMmzvfQr/RNAR+cOWkGwMI8ZoaTV57McvIeFwztY8szBKtyA==
  dependencies:
    "@mux/mux-data-theoplayer" "^4.17.1"

"@theoplayer/react-native-ui@^0.7.2":
  version "0.7.2"
  resolved "https://registry.yarnpkg.com/@theoplayer/react-native-ui/-/react-native-ui-0.7.2.tgz#1abefa0c9892da7da3fab44e449190404d585123"
  integrity sha512-Jdldj/5c/9E4cCzooUxhtGvISEQ9YQZHLLfd1F5d7EomRbVBztMwOY9FXkofexMoZ3ABP3cwM/YGxfyVOynEWg==
  dependencies:
    buffer "^6.0.3"
    react-native-svg-web "^1.0.0"
    react-native-url-polyfill "^1.3.0"
    url-polyfill "^1.1.12"

react-native-theoplayer@7.6.0:
  version "7.6.0"
  resolved "https://registry.yarnpkg.com/react-native-theoplayer/-/react-native-theoplayer-7.6.0.tgz#9bbb8e2269ea5dcfd51a40a279275fc36bd04055"
  integrity sha512-2/0TXkX7U5Dr9knLdj1g92284vBtvQp+CtNcXet3Q/6iZOpk2AUuHJRozzoRka/hL/2r7QwKWUMPfguJH8jm0A==
  dependencies:
    buffer "^6.0.3"
```

Podfile.lock:

```
PODS:
  - Mux-Stats-Core (3.14.0)
  - Mux-Stats-THEOplayer (0.7.0):
    - Mux-Stats-Core (~> 3.10)
  - react-native-theoplayer (7.6.0):
    - React-Core
    - THEOplayerSDK-core (~> 7.4)
  - react-native-theoplayer-mux (1.3.0):
    - Mux-Stats-THEOplayer (~> 0.7)
    - React-Core
    - react-native-theoplayer
    - THEOplayerSDK-core (7.6.1)
```

**Comment by wvanhaevre** [comment]
Thanks for raising this issue. We did an investigation and found a limitation on the Mux-Stats-THEOplayer's dependecies that prevents using recent THEOplayer versions. 
In collaboration with Mux, an update was released for Mux-Stats-THEOplayer: [v0.11](https://github.com/muxinc/mux-stats-sdk-theoplayer-ios/releases/tag/v0.11.0)
We are now upgrading our mux connector to depend on this latest release. Testing already proved that the crash is no longer happening with these more recent SDK's. We are now in the process of releasing a new version of our connector.
